AUTHORS

```diff
@@ -1,3 +1,4 @@
+Cliff Dyer <cliff@opencraft.com>
```

Comment:
Added my name at the top, since the repo was created from work I initially did on edx/edx-platform.